packages/grafana-ui/src/components/InlineToast/InlineToast.internal.story.tsx

```diff
@@ -33,7 +33,7 @@ export const InlineToast: Story<InlineToastProps> = (args) => {
   const [el, setEl] = useState<null | HTMLInputElement>(null);
 
   return (
-    <div>
+    <div style={{ maxWidth: 500, width: `calc(100% - 100px)` }}>
```

Comment:
It's not `style={}` that makes the component look odd in storybook but something else.

In main branch: 
<img width="289" alt="Screenshot 2023-09-20 at 09 03 23" src="https://github.com/grafana/grafana/assets/48948963/e38331c7-983a-4a27-8844-8361e7b6279d">

In this branch:
<img width="654" alt="Screenshot 2023-09-20 at 09 02 44" src="https://github.com/grafana/grafana/assets/48948963/575eab75-b495-4d2b-8217-8391a2c8fe27">

Reply:
This issue also appears for the AutoSaveField (which uses the InlineToast)

Reply:
I've run into this in the past with "pill" shaped elements. Pretty sure it's caused by the `border-radius: 100%` on the toast itself. Changing it from a percentage to a length like `100em` would probably solve it.

Reply:
@JoaoSilvaGrafana fixed this yesterday: https://github.com/grafana/grafana/pull/75098#event-10411326589

it's just not on this branch